airflow/www/views.py

```diff
@@ -2454,7 +2454,8 @@ def success(self):
     @action_logging
     def dag(self, dag_id):
         """Redirect to default DAG view."""
-        return redirect(url_for('Airflow.grid', dag_id=dag_id, **request.args))
+        kwargs = {**request.args, "dag_id": dag_id}
```

Comment:
it seems the effect here is not to prioritize the path, but to always ignore the query param... is that right?

Reply:
Oh, yes, that's correct. Let me reword the commit.